src/app/gps/qgsgpsinformationwidget.h

```diff
@@ -137,11 +137,13 @@ class APP_EXPORT QgsGpsInformationWidget: public QgsPanelWidget, public QgsMapCa
 
     void updateGpsDistanceStatusMessage();
 
+    void qualityDescription( int qualityIndicator );
```

Comment:
Could this be moved to the (core) class `QgsGpsInformation`? This would be useful outside this widget as well.